src/components/Menus/Sidebar/Sidebar.scss

```diff
@@ -4,6 +4,7 @@
 
 .sidebar {
   display: flex;
+  flex-shrink: 0;
```

Comment:
Removing overflow here does everything that's needed. Adding `flex-shrink` constraints makes the sidebar overflow when on mobile (see the close icon is off the screen, and scrollable to get there too!)

![image](https://github.com/RaspberryPiFoundation/editor-ui/assets/74183390/3c5729c6-7524-4d57-8c4d-799a7836b579)

```suggestion
```

Reply:
Ah yes, that fixed it, thanks 👍